SPECS/opentofu/opentofu.spec

```diff
@@ -0,0 +1,87 @@
+%bcond_without vendor
+
+Name:           opentofu
+Version:        1.6.2
+Release:        3%{?dist}
+Summary:        OpenTofu lets you declaratively manage your cloud infrastructure
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+
+License:        0BSD AND Apache-2.0 AND BSD-2-Clause AND BSD-3-Clause AND MIT AND MPL-2.0
+URL:            https://github.com/opentofu/opentofu
+Source0:        https://github.com/opentofu/opentofu/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+
+# Below is a manually created tarball, no download link.
```

Comment:
We have started a convention of adding a shell script that generates the vendor tarball automatically.
I think we might as well add that. 
E.G look at moby-engine/docker-cli etc etc

Reply:
+1, our automation relies on the `generate_source_tarball.sh` scripts with specific input arguments. There should be plenty of example to use for reference and they all have a very similar structure, so a copy-paste with minor tweaks should do the trick.

Reply:
I added a `generate_source_tarball.sh` for this package.